content/docs/reference/organizing-stacks-projects.md

```diff
@@ -96,22 +96,65 @@ and testing. In that case, we'll have six distinct stacks, that pair up together
 The way Pulumi programs communicate information for external consumption is by using stack exports. For instance,
 our infrastructure stack might export the Kubernetes configuration information needed to deploy into a cluster:
 
+{{< langchoose >}}
+
+```javascript
+exports.kubeConfig = ... a cluster's output property ...;
+```
+
 ```typescript
 export const kubeConfig = ... a cluster's output property ...;
 ```
 
+```python
+pulumi.export("kubeConfig", ... a cluster's output property ...)
+```
+
+```go
+// StackReference is not supported in Go currently.
```

Comment:
```suggestion
// StackReference is currently not supported in Go.
```

Reply:
JFWIW - This is copy pasted - I suspect there are lots more places we'll want to fix this up if it's worth changing.